projects/packages/search/src/dashboard/components/record-meter/record-count.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Returns record count component showing current records indexed and max records available for tier.
+ *
+ * @param {object} props - current record count and plan record limit.
+ * @returns {React.Component} record count component.
+ */
+export function RecordCount( props ) {
+	if ( ! props.recordCount || ! props.planRecordLimit ) {
+		return null;
+	}
+
+	return (
+		<div data-testid="record-count" className="jp-search-record-count">
+			{ props.recordCount && props.planRecordLimit && (
```

Comment:
Hi Anna, the check here is kinda duplicate to line 14. If we move the conditions before the outer element (div), then the two would be  replaceable. I think we could just use one?

Reply:
Ah yep, nice catch! I think you're right, thank you 👍